app/scenes/options_modal/options_modal_list.ios.js

```diff
@@ -10,24 +10,41 @@ import {
 } from 'react-native';
 import Font from 'react-native-vector-icons/FontAwesome';
 
+import FormattedText from 'app/components/formatted_text';
+
 export default class OptionsModalList extends PureComponent {
     static propTypes = {
         items: PropTypes.array.isRequired,
         onCancelPress: PropTypes.func,
-        title: PropTypes.string
+        title: PropTypes.oneOfType([
+            PropTypes.string,
+            PropTypes.object
+        ])
     }
 
     renderOptions = () => {
         const {items} = this.props;
 
         const options = items.map((item, index) => {
+            let textComponent;
+            if (typeof item.text === 'string') {
```

Comment:
Same as my last comment. I almost wonder if we should add that not-localized fallback behaviour to `FormattedText` although I don't think it'll work when using destructuring to pass in the props.